src/bptl/work_units/zgw/tasks/resultaat.py

```diff
@@ -18,10 +20,12 @@ class CreateResultaatTask(ZGWWorkUnit):
     **Required process variables**
 
     * ``zaakUrl``: full URL of the ZAAK to set the RESULTAAT for
-    * ``resultaattype``: full URL of the RESULTAATTYPE to set
     * ``bptlAppId``: the application ID of the app that caused this task to be executed.
       The app-specific credentials will be used for the API calls.
     * ``services``: DEPRECATED - support will be removed in 1.1
+    * ``resultaattype``: full URL of the RESULTAATTYPE to set
+    -OR-
+    * ``omschrijving``: description of RESULTAAT.
```

Comment:
*Please* check the documentation output on http://localhost:8000/taskmappings/add/, this is causing Sphinx errors now:

```
System Message: WARNING/2 (<string>, line 13)

Bullet list ends without a blank line; unexpected unindent.
```

Reply:
Good catch. Had checked it, but made some edits and apparently didn't check again.